### DISCUSSION TITLE
[Script request] Ghostfolio

### DISCUSSION THREAD

**Discussion by Dajestar** [community-scripts/ProxmoxVE#368] [Request script] [answered]
### Application Name

Ghostfolio

### Website

https://github.com/ghostfolio/ghostfolio

### Description

Ghostfolio is an open source wealth management software built with web technology. The application empowers busy people to keep track of stocks, ETFs or cryptocurrencies and make solid, data-driven investment decisions. The software is designed for personal use in continuous operation. 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by BoB5231** [comment]
To complete the set, it's worth adding a converter to easily import transactions from the broker:

**Website**
[https://github.com/dickwolff/Export-To-Ghostfolio](https://github.com/dickwolff/Export-To-Ghostfolio)

**Comment by fell-lucas** [comment]
Hey, I'll give it a try and see if I can write the script for Ghostfolio. Work in progress pull-request: https://github.com/community-scripts/ProxmoxVED/pull/798

The script works so far, I've tested it a few times. If you wish to run it before it gets merged into the repo, follow these steps:

1. Clone the [fork](https://github.com/fell-lucas/ProxmoxVED) and switch to the `add-ghostfolio` branch
2. Serve the root directory of the repo in a web server somehow - I prefer using [Bun](https://bun.sh/) and `serve` for this, running `bunx serve ./ -p 8000`
3. Replace line 2 of `ct/ghostfolio.sh` with `source <(curl -fsSL http://<your-machine-ip>:8000/misc/build.func)`
4. Also replace the curl URL in line 1441 of `misc/build.func` with `http://<your-machine-ip>:8000/install/${var_install}.sh`
5. Run the script as usual in the Proxmox host: `bash -c "$(curl -fsSL http://<your-machine-ip>:8000/ct/ghostfolio.sh)"`

**Reply by Dajestar** [reply]
Amazing, will try it out!

**Comment by drVLecter** [comment, marked as answer]
This one is available in main repo. 
Should maybe move it on the project board from Backlog to Done:

https://github.com/orgs/community-scripts/projects/9/views/1?pane=issue&itemId=87952108
